src/amo/components/AddonReview.js

```diff
@@ -19,40 +17,52 @@ export class AddonReviewBase extends React.Component {
     errorHandler: PropTypes.object.isRequired,
     i18n: PropTypes.object.isRequired,
     review: PropTypes.object.isRequired,
-    router: PropTypes.object.isRequired,
     updateReviewText: PropTypes.func,
   }
 
-  onSubmit = (event) => {
+  constructor(props) {
+    super(props);
+    this.state = { reviewTitle: null };
+    if (props.review) {
+      // TODO: also do this in componentWillReceiveProps.
+      this.state.reviewTitle = props.review.title;
+      this.state.reviewBody = props.review.body;
+    }
+    this.overlayCard = null;
+    this.reviewTextarea = null;
+    this.reviewTitleInput = null;
+  }
+
+  onSubmit = (event, { overlayCard = this.overlayCard } = {}) => {
+    const { reviewBody, reviewTitle } = this.state;
     event.preventDefault();
     event.stopPropagation();
-    const body = this.reviewTextarea.value;
     const params = {
-      body,
+      body: reviewBody,
+      title: reviewTitle,
       addonSlug: this.props.review.addonSlug,
       errorHandler: this.props.errorHandler,
       reviewId: this.props.review.id,
       apiState: this.props.apiState,
-      router: this.props.router,
     };
     // TODO: render a progress indicator in the UI.
```

Comment:
I think there's already an issue for this, you could add it to the comment if you like 😄

(I think it's https://github.com/mozilla/addons-frontend/issues/1156, right?)